_posts/Illumination_Correction_A_Comparison_of_Methods.md

```diff
@@ -0,0 +1,135 @@
+
+In this post, I created the pros and cons based on what I have researched and then my own personal experiences using these methods as seen in Figure 1.
+
+![Pros and Cons](analysis/Pros_and_Cons_Table.png)
+
+*Figure 1. Table of Illumination Correction Methods*
+
+**Let's go through each of these methods one by one:**
+
+I started off my research with CellProfiler (CP). 
+Whenever I searched for illumination correction methods, this software came up first. 
+The main way that it is used is through the GUI (graphical user interface).
+There are ways to utilize CP through Python and Jupyter notebooks (see [<u>https://github.com/CellProfiler/notebooks</u>](https://github.com/CellProfiler/notebooks)), but the most recent examples are from 2-5 years ago.
+
+***Side-note:*** If you aren’t familiar with the coding world, it is important to know that software changes frequently and can become out of date even after less than a year. 
+Please use that resource I provided with a grain of salt as it very well could be out of date or work perfectly!
+
+Back to the GUI, CP 4.0 has a very user-friendly interface that makes it easy for non-data science biologists to use as well as data science beginners. 
+But this comes with a not-so-fun challenge that impacts generalizability, which is manual parameters (I know, terrifying!).
+Now, a few manual parameters are fine within a software, like if you need to make minor corrections to fit the function to your data, but too many manual parameters creates many avenues for confusion and error. 
+When it comes to CP illumination correction, it has MANY manual parameters (ranging from 5 to 15 depending on the method you choose).
+As a beginner in the field, how are you meant to know which threshold to use or if the correction should be run based on all images or per image? 
+That is my main issue with CP. It is set up to be easy to use for beginners, but you need to be an expert to use it properly.
+
+There is [documentation](https://cellprofiler-manual.s3.amazonaws.com/CellProfiler-4.2.1/index.html) on the modules and parameters that can help with trying to understand the IC function you are creating. 
+The things that CP does better than the other two methods are that it makes loading in and downloading the corrected images easy, and you can test-run your pipeline to make corrections. 
+It is nice when software has its own method of saving instead of having to troubleshoot.
+As well, CP works like a Jupyter Notebook where you can test each module and figure out any issues instead of running the full pipelines. 
+I believe this software can be used effectively when you put a lot of time into research and investigate as many examples as possible to find the best combination of parameters. 
+But I also believe that after all the time I put into understanding CP, I am still left with doubt in all my
+pipelines and don’t feel comfortable using them for my current project.
```

Comment:
```suggestion

I believe this software can be used effectively when you put a lot of time into research and investigate as many examples as possible to find the best combination of parameters. 
But I also believe that after all the time I put into understanding CP, I am still left with doubt in all my pipelines and don’t feel comfortable using them for my current project (predicting NF1 genotype from Schwann Cell morphology).
```

Reply:
it looks like a lot changed, but I just added a paragraph and fixed a premature line break

Reply:
I also added a brief description of your project (see comment below!)